src/components/Dialog/Modal.tsx

```diff
@@ -93,15 +100,15 @@ export const Modal = ({
                 <section
                     className={
                         `${eccgui}-dialog__wrapper` +
-                        (typeof size === 'string' ? ` ${eccgui}-dialog__wrapper--` + size : '') +
-                        (className ? ' ' + className : '')
+                        (typeof size === "string" ? ` ${eccgui}-dialog__wrapper--` + size : "") +
+                        (className ? " " + className : "")
```

Comment:
Same here. What is the simplification in this case? Looks still the same in slightly different syntax. Is it better to read?

Reply:
I won't change this one because I'd say it is harder to read. (Btw, the change came in by auto linting, the code itself does not really changed for this feature branch)